Makefile

```diff
@@ -2,10 +2,15 @@
 
 up:
 	minikube start
+	minikube addons enable ingress
+	minikube addons enable dashboard
 	kubectl create namespace gitlab
 	helm repo add gitlab https://charts.gitlab.io
 	helm repo update
-	helm install gitlab gitlab/gitlab --namespace gitlab --values values.yaml --version v7.1.0
```

Comment:
The pinned version is important for reproducible build.  Is there a reason to drop this and use the latest version every time it is used?

Reply:
Just so we're not stuck in some old version:) I brought it back